app/controllers/stats_controller.rb

```diff
@@ -0,0 +1,43 @@
+
+
+require_relative 'stats_helpers/donations.rb'
+require_relative 'stats_helpers/stats_html.rb'
+
+class StatsController < ApplicationController
+    def donation_totals
+        @result = ActiveRecord::Base.connection.execute($donation_query)
```

Comment:
Is there no way to do this via AR?

Reply:
the query is a complex copy paste of our seller payout queries, because its a query between multiple tables and entities im not sure how to express this performantly that wouldn't end up in some bottle neck or not able to be reproducable . 

https://data.heroku.com/auth/heroku/callback?code=03140850-8314-44ab-9303-3a966b46770f